ec2-spot-terraform/.gitignore

```diff
@@ -0,0 +1,6 @@
+terra*
+.terraform/*
+<<<<<<< HEAD
```

Comment:
This .gitignore seems to come from a patch or a broken merge that you had. Please fix